relay-event-normalization/src/event.rs

```diff
@@ -2852,6 +2862,97 @@ mod tests {
         insta::assert_ron_snapshot!(SerializableAnnotated(&Annotated::new(event)), {});
     }
 
+    #[test]
+    fn test_computes_performance_score_and_tags_with_profile_version() {
+        let json = r#"
+        {
+            "type": "transaction",
+            "timestamp": "2021-04-26T08:00:05+0100",
+            "start_timestamp": "2021-04-26T08:00:00+0100",
+            "measurements": {
+                "cls": {"value": 0.11},
+                "inp": {"value": 120.0}
+            }
```

Comment:
Does the minimal test require two measurements?

Reply:
I've modified this to be a bit more minimal. Now there's only one profile and just a single measurement to drive a calculation result.

https://github.com/getsentry/relay/pull/3249/commits/834aa616c79c48e9e2744774c648f330ab9215d2